scripts/start.sh

```diff
@@ -78,7 +78,7 @@ docker-compose up -d schemaregistry connect control-center
 
 echo
 echo -e "Create topics in Kafka cluster:"
-docker-compose exec kafka1 bash -c "/tmp/helper/create-topics.sh" || exit 1
+docker-compose exec tools bash -c "/tmp/helper/create-topics.sh" || exit 1
```

Comment:
This is UBI-image-tools-restriction-driven ... `kafka1` doesn't have `xargs` (or `find` ...), whereas `tools` does.  I used `xargs` in the certificate-generation parallelisation, but that was on the host - tried to use the same approach here, but `xargs` availability is limited.  Could probably employ a bash parallelisation approach that doesn't use `xargs`, but it does do it in a tidy way.

Reply:
Thanks for explaining context.  Since `tools` won't work in a few weeks/months, I guess this needs `xargs` on host solution.

Reply:
In this case I don't think `xargs` on host will be good ... we can try, but this is different from certificate-generation, where it is all done on the host ... since we don't assume Kafka client on host (nor shouldn't) the parallelisation is going to involve multiple `exec`s into Docker containers.

We should explorer non-`xargs` in-container parallelisation using `bash`-native.